### PR TITLE
print solver logs

### LINKED ISSUE #1709: Access to optimization solver logs
**Description**

It seems that there is no way in the simulator, today, to get the logs of the underlying solver (xpress through ortools, for example).

This would be highly useful to get information about the execution and performances of the optimization problem resolution.

Mentioned by @tbittar. 

Note that antares-xpansion already provides this possibility: if needed, we can get some implementation guidance from there, although it might not be relevant when using it through ortools.

### PR DESCRIPTION
**Pull request by a-zakir**
In this Pr, i'm trying to enable solvers logs:
##TODO
- [x] with ortools interface
- [x] Sirius direct call
added : 
 - CLI option ` --solver-logs`
 - `solver-log` boolean option to be read in `generaldata.ini`
will close #1709 maybe #1427?

### CODE REVIEW

**Review by payetvin — APPROVED** [review]

File: docs/reference-guide/10-command_line.md [hunk 1]
```diff
@@ -40,6 +40,7 @@ _In all cases, arguments " –h" or "–help" can be used to get help_
 |--no-ts-import | Do not import timeseries into the input folder. <br/> (This option may be useful for running old studies without upgrade)|
 |-m, --mps-export | Export anonymous mps weekly or daily optimal UC+dispatch linear |
 |-s, --named-mps-problems | Export named mps weekly or daily optimal UC+dispatch linear |
+|-o, --solvers-logs | Print solver logs |
```
Comment (sylvlecl) [inline docs/reference-guide/10-command_line.md]: Would prefer "solver-logs" without the plural.

Also, I would remove the short option, I think it's OK to only have the long one.
We can keep the "-o" for a future option that would be more compatible with it.

**Review by sylvlecl — CHANGES_REQUESTED** [review]

File: docs/reference-guide/10-command_line.md (hunk 1 above)
Comment (a-zakir) [reply docs/reference-guide/10-command_line.md]: good

Comment (a-zakir) [reply docs/reference-guide/10-command_line.md]: done!

File: src/libs/antares/study/parameters.cpp [hunk 2]
```diff
@@ -626,6 +627,11 @@ static bool SGDIntLoadFamily_Optimization(Parameters& d,
     {
         return stringToGlobalTransmissionCapacities(value, d.transmissionCapacities);
     }
+
+    if (key == "solver-logs")
```
Comment (sylvlecl) [inline src/libs/antares/study/parameters.cpp:631]: We'll need to add it in the documentation too (new section v8.9 in the file format doc).

Comment (a-zakir) [reply src/libs/antares/study/parameters.cpp:631]: was thinking about it,  and said to myself that @flomnes  would do it 😛

**Review by sylvlecl — APPROVED** [review]

**Pull request merged**